Source/WebCore/testing/Internals.h

```diff
@@ -409,6 +409,7 @@ class Internals final : public RefCounted<Internals>, private ContextDestruction
 
     ExceptionOr<unsigned> wheelEventHandlerCount();
     ExceptionOr<unsigned> touchEventHandlerCount();
+    ExceptionOr<unsigned> scrollableAreaSize(Node&);
```

Comment:
"area size" is ambiguous. Is it a size, or an area? Returning an area doesn't seem like a great way to tell if the size changed (height and width could swap, for example).

Reply:
Changed this to return just the width.